ReClass.NET/app.manifest

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't like that the manifest forces ReClass.NET to run as admin because that's not always necessary. Maybe it would be possible to add this as a setting which forces windows to run it as admin. (Like in the file property dialog where you can choose to always run a program with higher privileges.)

Reply:
Or ReClass.NET could restart itself with WinUtil.RunElevated if requested.

Reply:
Yeah that's probably a lot better